boa/src/builtins/value/mod.rs

```diff
@@ -374,6 +370,8 @@ impl ValueData {
         }
     }
 
+    // pub fn to_string(&self, ctx: &mut Interpreter) {}
+
```

Comment:
What is happening here? Should we add a `TODO` or should it be removed?

Reply:
I wasn't sure wether to add it to Value of Interpreter. I removed it.